Packs/CommonTypes/IndicatorTypes/reputation-url.json

```diff
@@ -7,7 +7,7 @@
   "commitMessage": "",
   "shouldPublish": false,
   "shouldCommit": false,
-  "regex": "(?i)(?:(?:https?|ftp|hxxps?):(\\/?\\/?\\\\?\\\\?)|www\\[?\\.\\]?|ftp\\[?\\.\\]?)(?:[-_\\d\\p{L}]+\\[?\\.\\]?)+[-_\\d\\p{L}]+(?::\\d+)?(?:(?:((\\/)|(\\*)|(\\#)|(\\?)))+(?:[-\\d\\p{L}+&@'#\\/%=~_$!\\-:,.;\\\\*–—―|]*|\\([-\\d\\p{L}\\+&@#\\/%=~_\\$\\?!\\-:,\\.;\\\\\\*–—―\\|]*\\))*)*",
+  "regex": "(?i)((?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])(?:\\[\\.\\]|\\.)){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])(\\/)|(?:(?:https?|ftp|hxxps?):(\\/?\\/?\\\\?\\\\?)|www\\[?\\.\\]?|ftp\\[?\\.\\]?))(?:[-_\\d\\p{L}]+\\[?\\.\\]?)+[-_\\d\\p{L}]+(?::\\d+)?(?:(?:((\\/)|(\\*)|(\\#)|(\\?)))+(?:[-\\d\\p{L}+&@'#\\/%=~_$!\\-:,.;\\\\*–—―|]*|\\([-\\d\\p{L}\\+&@#\\/%=~_\\$\\?!\\-:,\\.;\\\\\\*–—―\\|]*\\))*)*",
```

Comment:
I assume the slash should not be in a matched group.
are you intentionally supporting both `1.1.1.1` and `1[.]1[.]1[.]1`?
```suggestion
  "regex": "(?i)((?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])(?:\\[\\.\\]|\\.)){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])/|(?:(?:https?|ftp|hxxps?):(\\/?\\/?\\\\?\\\\?)|www\\[?\\.\\]?|ftp\\[?\\.\\]?))(?:[-_\\d\\p{L}]+\\[?\\.\\]?)+[-_\\d\\p{L}]+(?::\\d+)?(?:(?:((\\/)|(\\*)|(\\#)|(\\?)))+(?:[-\\d\\p{L}+&@'#\\/%=~_$!\\-:,.;\\\\*–—―|]*|\\([-\\d\\p{L}\\+&@#\\/%=~_\\$\\?!\\-:,\\.;\\\\\\*–—―\\|]*\\))*)*",
```